frontend/Utils/flaxFetch.tsx

```diff
@@ -15,15 +15,9 @@ export function flaxFetch<ResponseDataType = object>(
     if (r.ok) {
       if (r.status === 204) {
         return;
-      } else if (
-        r.headers["content-type"] &&
```

Comment:
This was a bug in flaxFetch - `r.headers['content-type']` was never defined in Chrome.